### PR TITLE
Pipe+morph

### PR DESCRIPTION
**Pull request by behrica**
proposal to separate pipeline and metamorph ns

### CODE REVIEW

File: src/tablecloth/pipeline.clj [hunk 1]
```diff
@@ -1,8 +1,7 @@
 (ns tablecloth.pipeline
```
Comment (behrica) [inline src/tablecloth/pipeline.clj]: I copied here the old code from pipeline.clj

**Comment by genmeblog** [comment]
I was thinking about that and imho keeping two, almost the same implementations (one without wrapping, second with) can cause high confusion. Why to have both? 
I think it's safe to provide breaking change and keep only metamorph path. User always can extract dataset from the result map.

**Comment by behrica** [comment]
Fine for me.
So then we keep methamorph only .

I would propose to call the ns 'metamorph' instead of pipeline,  this is then consistent with dataset.

**Comment by genmeblog** [comment]
I would stay with pipeline. One reason is it's about the pipeline (metamorph is just a library name which helps with running pipelines). Second reason is that it would be bigger breaking change, than just a map (new) instead of dataset (old)

**Comment by behrica** [comment]
fine for me.
I suppose this PR can be closed then and you will just merge as is ?

**Comment by genmeblog** [comment]
Yes, it can be closed. I will update tablecloth tomorrow. Thanks for the effort anyway.

**Pull request closed without merging**